tests/foreman/cli/test_satellitesync.py

```diff
@@ -2541,6 +2541,8 @@ def test_positive_custom_cdn_with_credential(
 
     @pytest.mark.e2e
     @pytest.mark.tier3
+    @pytest.mark.pit_server
+    @pytest.mark.pit_client
     @pytest.mark.rhel_ver_list([8])
```

Comment:
Since RHEL 8 reached its last `Y` version, this test will never be picked up in **PIT client** testing as it is pinned to RHEL 8. I recommend making the test compatible with RHEL 9+ or removing the `pit_client` marker.

Reply:
I'll pin it to the `settings.content_host.default_rhel_version`, it seems to be `9` already.